src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BlockMetalLadder.java

```diff
@@ -103,11 +104,11 @@ public AxisAlignedBB getBoundingBox(IBlockState state, IBlockAccess world, Block
 	{
 		if(this.getMetaFromState(state)==0)
 		{
-			TileEntityLadder tile = (TileEntityLadder)world.getTileEntity(pos);
-			if(tile!=null)
+			TileEntity tileEntity = world.getTileEntity(pos);
+			if(tileEntity!=null)
```

Comment:
This should probably be an `instanceof` check.

Reply:
I'm taking about checking whether the TE extends `IEBlockInterfaces.IDirectionalTile`. If it doesn't but isn't null it will crash in line 110.